account_banking_payment_export/wizard/payment_order_create.py

```diff
@@ -83,7 +80,7 @@ def filter_lines(self, lines):
         """
         self.ensure_one()
         payment_lines = self.env['payment.line'].\
-            search([('order_id.state', 'in', ('draft', 'open')),
+            search([('order_id.state', '!=', 'cancel'),
                     ('move_line_id', 'in', lines.ids)])
```

Comment:
I'm not sure. What if, for instance, a payment order is sent/done, but for some reason (partially) rejected by the bank. The user will unreconcile some move lines and expect to be able to add the move line to another payment order. For reference, this domain has been defined after discussion in #110, #93 and related issues and PR. So I'd say :-1: but requires further thinking.